dev/archery/archery/integration/datagen.py

```diff
@@ -1075,12 +1095,27 @@ def _temp_path():
         generate_primitive_case([], name='primitive_no_batches'),
         generate_primitive_case([17, 20], name='primitive'),
         generate_primitive_case([0, 0, 0], name='primitive_zerolength'),
-        generate_decimal_case(),
+
+        generate_null_case([10, 0])
+        .skip_category('JS')
+        .skip_category('Go'),
+
+        # TODO(ARROW-3676): Go tests
+        generate_decimal_case().skip_category('Go'),
```

Comment:
ARROW-3676 is resolved

```suggestion
        generate_decimal_case(),
```

Reply:
Unfortunately, tests fail: [ARROW-7948](https://issues.apache.org/jira/browse/ARROW-7948)